AnnoyancesFilter/sections/popups.txt

```diff
@@ -124,6 +124,7 @@ theoryandpractice.ru##a[href^="https://form.typeform.com/"]
 !
 mail.yahoo.co.jp#?#.main > div[class] > svg + div[class] > a[title=""][font-size="14"]:upward(2)
 video.dmkt-sp.jp##.lp_cta_fixed
+flaschenpost.de##.fp_modal
```

Comment:
Same popup is used for delivery address check
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/170124197-1778fad7-964f-4e5f-a98a-11095e0f9ce1.png)

</details>